test/mixins.test.js

```diff
@@ -79,11 +79,24 @@ describe('Model class', function() {
     properties.city.should.eql({ type: String, required: true });
   });
 
+  it('should fail to apply an undefined mixin class', function(done) {
+    var memory = new DataSource('mem', { connector: Memory }, modelBuilder);
+    function applyMixin() {
+      var UndefinedMixin;
```

Comment:
If you're referring to the UndefinedMixin, that's used, otherwise please clarify. Thanks